project/Build.scala

```diff
@@ -38,7 +38,20 @@ import org.scalajs.core.tools.linker.backend.OutputMode
 
 import sbtassembly.AssemblyPlugin.autoImport._
 
-object Build extends sbt.Build {
+/*
+ * In sbt 0.13 the Build trait would expose all vals to the shell, where you can use them in "set a := b"
+ * like expressions. This re-exposes them.
+ */
+object ExposedValues extends AutoPlugin {
+  override def requires = plugins.JvmPlugin
+  override def trigger = allRequirements
```

Comment:
I don't think the trigger is necessary, nor is `requires`. We don't need the plugin to be applied on any project. We just want its `autoImport`s.